webpack.config.js

```diff
@@ -67,7 +68,7 @@ module.exports = {
       //NOTE: Bootstrap css configuration
       {
         test: /\.css$/,
-        include: /node_modules\\bootstrap/,
```

Comment:
This is the change that I commented last day, @brauliodiez , we don't need to specify bootstrap folder, only we need `node_modules`, because we are only loading _.css_ files allowed in vendorStyles property